### DISCUSSION TITLE
What are the features for Readarr and SABnzbd scripts?

### DISCUSSION THREAD

**Discussion by Klomgor** [RandomNinjaAtk/arr-scripts#113] [Q&A] [answered]
I don't use Radarr or Sonarr scripts because I prefer hotio's images which are not Alpine based, but SABnzbd and Readarr scripts don't have any features listed in their descriptions, and when I tried them I saw very similar output in the logs to the Lidarr script and the same /config/extended.conf.

Am I missing something here?

**Comment by RandomNinjaAtk** [comment, marked as answer]
WIki is open for edits, limited functionality was added to each and it's largely the same functionality as the old containers with some slight tweaks for this new installation method:

https://github.com/RandomNinjaAtk/docker-sabnzbd-extended
https://github.com/RandomNinjaAtk/docker-readarr-extended
